### PR TITLE
[docs] Fixes to homepage, docs front page, sphinx conf

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: hail/python/hail/docs/index.rst [hunk 1]
```diff
@@ -1,8 +1,11 @@
-.. Hail documentation master file, created by
-   sphinx-quickstart on Fri Nov  4 10:55:10 2016.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+========
+Hail 0.2
+========
 
+Hail is an open-source library for scalable data exploration and analysis, with
+a particular emphasis on genomics. See the `overview <overview/index.html>`_ for a
```
Comment (jbloom22) [inline hail/python/hail/docs/index.rst]: "a particular emphasis" => "extensive support for"?

File: hail/python/hail/docs/index.rst [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+high-level walkthrough of the library, or the `installation <getting_started.html>`_
```
Comment (jbloom22) [inline hail/python/hail/docs/index.rst]: How about:

See the `overview <overview/index.html>`_ for a high-level walkthrough of the library, the `GWAS tutorial <...>` for a simple genomics example, and the `installation page <getting_started.html>`_ to get started.

File: README.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Hail uses a continuous deployment approach to software development, which means
 
 ### Maintainer
 
-Hail is maintained by a team in the [Neale lab](https://nealelab.squarespace.com/) at the [Stanley Center for Psychiatric Research](http://www.broadinstitute.org/scientific-community/science/programs/psychiatric-disease/stanley-center-psychiatric-research/stanle) of the [Broad Institute of MIT and Harvard](http://www.broadinstitute.org) and the [Analytic and Translational Genetics Unit](https://www.atgu.mgh.harvard.edu/) of [Massachusetts General Hospital](http://www.massgeneral.org/).
+Hail is maintained by a team in the [Neale lab](https://nealelab.is/) at the [Stanley Center for Psychiatric Research](http://www.broadinstitute.org/scientific-community/science/programs/psychiatric-disease/stanley-center-psychiatric-research/stanle) of the [Broad Institute of MIT and Harvard](http://www.broadinstitute.org) and the [Analytic and Translational Genetics Unit](https://www.atgu.mgh.harvard.edu/) of [Massachusetts General Hospital](http://www.massgeneral.org/).
```
Comment (jbloom22) [inline README.md]: Also, the Stanley URL can be shortened to "https://www.broadinstitute.org/stanley"

File: hail/python/hail/docs/index.rst (hunk 1 above)
Comment (tpoterba) [reply hail/python/hail/docs/index.rst]: I prefer the emphasis one

**Pull request merged**